cmd/ncproxy/main.go

```diff
@@ -64,7 +64,7 @@ func main() {
 
 	// If there's a node network service in the config, assign this to our global client.
 	if conf.NodeNetSvcAddr != "" {
-		log.G(ctx).Debugf("connecting to NodeNetworkService at address %s", conf.NodeNetSvcAddr)
+		log.G(ctx).Infof("connecting to NodeNetworkService at address %s", conf.NodeNetSvcAddr)
```

Comment:
was there a specific reason to change these?

Reply:
Felt more of an info level log

Reply:
and to match with the others in the file